src/fixtures/valid/valid.txt

```diff
@@ -252,14 +252,7 @@ mds_connect               44 samples [usec] 10 1774 4146 3299378
 mds_statfs                64446 samples [usec] 6 87 1723266 50620388
 obd_ping                  4 samples [usec] 10 16 50 644
 out_update                23 samples [usec] 7 118 879 59435
-ost.OSS.ost_seq.stats=
```

Comment:
Instead of removing valid data, add a new entry that shows new behavior.

Reply:
I added a new fixture and kept the existing one intact